src/pages/AddItem/AddItemPage.js

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import FileInput from "./FileInput";
+import Tag from "./Tag";
+import * as S from "./Styles/AddItemPageStyles";
+
+function AddItemPage(props) {
+  const [value, setValue] = useState({
+    imgFile: null,
+    name: "",
+    introduce: "",
+    price: "",
+    tag: "",
+  });
+  const [tags, setTags] = useState([]);
+
+  const handleChange = (name, value) => {
+    setValue((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    handleChange(name, value);
+  };
+
+  const isFormValid = () => {
+    const { name, introduce, price, tag } = value;
+    return (
+      name.trim() !== "" &&
+      introduce.trim() !== "" &&
+      price.trim() !== "" &&
+      tag.trim() !== ""
+    );
+  };
+
+  // Tags
+  const AddTags = (e) => {
+    const inputValue = e.target.value;
+
+    if (e.key === "Enter" && inputValue !== "" && !tags.includes(inputValue)) {
+      setTags([...tags, inputValue]);
+      e.target.value = "";
+    }
+  };
+
+  return (
+    <S.Wrapper>
+      <S.Header>
+        <S.HeaderTitle>상품 등록하기</S.HeaderTitle>
+        <S.SubmitButton type="button" disabled={isFormValid()}>
+          등록
+        </S.SubmitButton>
+      </S.Header>
+      <S.Main>
+        <div>
+          <S.Title>상품 이미지</S.Title>
+          <FileInput
+            name="imgFile"
+            value={value.imgFile}
+            onChange={handleChange}
+          />
+        </div>
+
+        <div>
+          <S.Title>상품명</S.Title>
+          <S.Input
+            name="name"
+            placeholder="상품명을 입력해주세요."
+            onChange={handleInputChange}
+          />
+        </div>
```

Comment:
똑같은 구조로 반복 사용되는 코드가 있네요!
이런 경우 컴포넌트를 분리해보는것도 나중에 유지보수할때 좋겠죠? :)
```suggestion
      <InputContainer {...props} />
```